tests/react/vanilla-utils/freezeAtom.test.tsx

```diff
@@ -35,6 +35,58 @@ it('freezeAtom basic test', async () => {
   await screen.findByText('isFrozen: true')
 })
 
+it('freezeAtom handles null correctly', async () => {
+  const nullAtom = atom(null, (_get, set, _arg: null) => {
+    set(nullAtom, null)
+  })
+
+  const Component = () => {
+    const [value, setValue] = useAtom(freezeAtom(nullAtom))
+    return (
+      <>
+        <button onClick={() => setValue(null)}>set null</button>
+        <div>value is null: {`${value === null}`}</div>
+      </>
+    )
+  }
+
+  render(
+    <StrictMode>
+      <Component />
+    </StrictMode>,
+  )
+
+  await screen.findByText('value is null: true')
+})
+
+it('freezeAtom handles primitive correctly', async () => {
+  const numberAtom = atom(123, (_get, set, _arg: number) => {
+    set(numberAtom, 456)
+  })
+
+  const Component = () => {
+    const [value, setValue] = useAtom(freezeAtom(numberAtom))
+    return (
+      <>
+        <button onClick={() => setValue(456)}>set number</button>
```

Comment:
```suggestion
  const numberAtom = atom(123, (_get, set, _ignored: number) => {
    set(numberAtom, 456)
  })

  const Component = () => {
    const [value, setValue] = useAtom(freezeAtom(numberAtom))
    return (
      <>
        <button onClick={setValue}>set number</button>
```
@dai-shi 

How about aligning the format with the existing test code for consistency?